modules/search/examples/autocomplete-search-index.jsonc

```diff
@@ -1,10 +1,8 @@
 {
     "type": "fulltext-index",
-    "name": "e-ngram-2-8",
-    "uuid": "$GENERATED_INDEX_UUID",
+    "name": "$INDEX_NAME",
     "sourceType": "gocbcore",
     "sourceName": "$BUCKET_NAME",
-    "sourceUUID": "$BUCKET_UUID",
```

Comment:
Does the sourceUUID and UUID need to be deleted for every index example? I'm confused why you deleted it for this one and not the others.

Reply:
In short yes if you want to create an index.

If you use the REST API the UUID specifies that this is an update to an existing index - thus if you want to create an index you must delete this value.  The self managed UI requires a delete, Eben might be actually "strippng this" in his newer UI's fo ELixir and Capella V2. 

Next sourceUUID is locked to a "bucket" so if you are trying to create a new index on the same cluster it could stay in BUT if you are moving the definition to a new cluster the sourceUUID will not match the bucket name and a create index will fail.